src/zepben/evolve/streaming/grpc/grpc_channel_builder.py

```diff
@@ -164,3 +164,18 @@ def with_token_fetcher(self, token_fetcher: ZepbenTokenFetcher) -> 'GrpcChannelB
             grpc.metadata_call_credentials(AuthTokenPlugin(token_fetcher=token_fetcher))
         )
         return self
+
+    def with_client_token(self, client_token: str) -> 'GrpcChannelBuilder':
+        """
+        Authenticates calls for the gRPC channel using a client access token string.
+
+        :param client_token: The :class:`str` that has been generated in Evolve application.
+        :return: This builder
+        """
+        if client_token is None:
+            raise Exception("You must have a client_token.")
```

Comment:
i don't know if i have to check `self._channel_credentials` like it was done in `with_token_fetcher` to check if the `make_secure` was called.

Reply:
That is correct